filter_finalreport.py

```diff
@@ -0,0 +1,47 @@
+#! /usr/bin/env python3
+
+import argparse
+import glob
+
+
+def slice_file(input_file, chromosomes):
+    write_file = open("{}_sliced.txt".format(input_file[:-4]), "w")
+    input_lines = open(input_file, "r")
+    chrom_list = chromosomes.split(",")
+
+    """ Write all lines until Data section """
+    line = input_lines.readline()
+    while line.rstrip() != '[Data]':
+        write_file.write(line)
+        line = input_lines.readline()
+    write_file.write(line)  # Print [Data] line
+
+    """ Index header line for chromosome column, and write header"""
+    header = input_lines.readline()
+    chromosome_index = header.split("\t").index("Chr")
+    write_file.write(header)
+
+    """ Write all variant lines are on chromosomes of interest"""
+    for line in input_lines:
+        splitline = line.rstrip().split("\t")
+        if splitline[chromosome_index] in chrom_list:
+            write_file.write(line)
+
+    write_file.close()
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--input_file', help='Full path to input file (must be a FinalReport.txt file)')
+    parser.add_argument(
+        '--chromosomes',
+        default="13,18,21,X,Y,XY",
+        help='comma seperated chromosomes to be kept in file. (default= 13,18,21,X,Y,XY)'
+    )
+    parser.set_defaults(func=slice_file)
+    args = parser.parse_args()
+    if not args.input_file:
+        for input_file in glob.glob("*FinalReport.txt"):
+            args.func(input_file, args.chromosomes)
```

Comment:
is it clear to the user that this file is taken by default? normally if you dont add an argument, you get the help page, but here everything is default. So you might want to make a --help/-h page or say something in the readme about the defaults?

Reply:
This is the akward thing of running it in production as a Windows binairy. 
Binairy will be copied into a folder including FinalReport.txt file(s) and started to make the filtered files.
No options needed, just need to do one thing without informing the (windows) user.

Optional arguments are only if we like to use the script in a Linux environment actually.

Reply:
after small discussion in Teams about this, approved.
this will be an executable on windows, and the argparse is only set-up this way, so that we can change it accordingly when we run it ourselves on linux. 👍